src/WebJobs.Script.WebHost/WebJobsApplicationBuilderExtension.cs

```diff
@@ -27,6 +33,10 @@ public static IApplicationBuilder UseWebJobsScriptHost(this IApplicationBuilder
             IOptionsMonitor<HttpBodyControlOptions> httpBodyControlOptionsMonitor = builder.ApplicationServices.GetService<IOptionsMonitor<HttpBodyControlOptions>>();
             IServiceProvider serviceProvider = builder.ApplicationServices;
 
+            // Update the UseWebJobsScriptHost method with the following
+            _environment = environment;
+            _placeholderTimer = new Timer(OnTimedEvent, null, 60000, 60000);
```

Comment:
This change seems unrelated? Did you mean to include this? We already have some code do this in StandByManager class. rite?

Reply:
Ah this was for testing some placeholder things manually - will remove it.